### PR TITLE
Remove categories mapping

### LINKED ISSUE #871: Remove category mapping
The mapping is managed on the store now. This code can be removed now:

https://github.com/canonical-web-and-design/charmhub.io/blob/master/webapp/store/logic.py#L53-L104

### PR DESCRIPTION
**Pull request by solazio**
## Done
- _Remove categories mapping._

## How to QA
- Visit https://charmhub-io-901.demos.haus
- See that everything works as expected when you change filters or search for charms

**Do not merge it before the store categories change is landed!!**

## Issue / Card
Fixes #871

## Screenshots
[if relevant, include a screenshot]

### CODE REVIEW

File: webapp/store/views.py [hunk 1]
```diff
@@ -38,16 +38,14 @@
 ]
 
 CATEGORIES = [
-    {"slug": "ai/ml", "name": "AI/ML"},
+    {"slug": "ai-ml", "name": "AI/ML"},
     {"slug": "big-data", "name": "Big Data"},
-    {"slug": "database", "name": "Database"},
     {"slug": "cloud", "name": "Cloud"},
-    {"slug": "containers", "name": "Containers"},
```
Comment (tbille) [inline webapp/store/views.py:45]: why removing the containers category?

File: webapp/store/views.py [hunk 2]
```diff
@@ -38,16 +38,14 @@
...
-    {"slug": "featured", "name": "Featured"},
-    {"slug": "logging-and-tracing", "name": "Logging and Tracing"},
+    {"slug": "database", "name": "Database"},
+    {"slug": "kubernetes", "name": "Kubernetes"},
```
Comment (tbille) [inline webapp/store/views.py:45]: was this new category agreed somewhere?

Comment (tbille) [reply webapp/store/views.py:45]: ah I got my answer. I opened a bug on launchpad. we should keep what we had: https://bugs.launchpad.net/snapstore-server/+bug/1921382

Comment (solazio) [reply webapp/store/views.py:45]: ahh, ok. I'll revert that part.

**Review by jkfran — APPROVED** [review]
LGTM

**Pull request merged**